app/Models/Concerns/HasFilters.php

```diff
@@ -483,6 +456,18 @@ protected function filterRace(Builder $query, string $value = null): void
             $query->whereRaw('(select count(*) from character_race as cr where cr.character_id = ' .
                 $this->getTable() . '.id and cr.race_id = ' . ((int) $value) . ') = 0');
             return;
+        } elseif ($this->filterOption('children')) {
```

Comment:
There is a lot of duplicated code here between the children option and the default option. Can you think of a way of merging it together and avoid duplicate code? For races, families, and orgs.